vfs/vfs.go

```diff
@@ -34,11 +34,18 @@ const DefaultContentType = "application/octet-stream"
 // ForbiddenFilenameChars is the list of forbidden characters in a filename.
 const ForbiddenFilenameChars = "/\x00"
 
-// RootFolderID is the identifier of the root folder
-const RootFolderID = "io.cozy.files.rootdir"
-
-// FsDocType is document type
-const FsDocType = "io.cozy.files"
+const (
+	// FsDocType is document type
+	FsDocType = "io.cozy.files"
+	// RootFolderID is the identifier of the root directory
+	RootFolderID = "io.cozy.files.rootdir"
+	// TrashFolderID is the identifier of the trash directory
+	TrashFolderID = "io.cozy.files.trashdir"
+	// TrashDirName is the path of the trash directory
+	TrashDirName = "/.cozy_trash"
+	// AppsDirName is the path of the directory in which apps are stored
+	AppsDirName = "/.cozy_apps"
```

Comment:
In the documentation, it's `/Apps`. We should fix the code or the documentation to be coherent